test/recipes/80-test_cmp_http.t

```diff
@@ -287,7 +296,8 @@ sub start_mock_server {
         stop_mock_server($pid);
         return 0;
     }
-    $server_tls = $kur_port = $pbm_port = $server_port;
+    $kur_port = $pbm_port = $server_port;
+    $server_tls = $server_port if $server_tls > 0;
```

Comment:
That doesn't look right ... X = Y if X > 0 ...

Reply:
Yeah, it looks weird, 
but we use the script in contexts where `$server_tls` may already have a real value, or `0` to indicate that TLS must not be used. In the latter case `$server_tls` must not be updated here.